presto-docs/src/main/sphinx/admin/properties.rst

```diff
@@ -989,3 +989,16 @@ Logging Properties
     * **Default value:** ``100MB``
 
     The maximum file size for the log file of the HTTP server.
+
+
+Legacy Compatible Properties
+------------------------------
+
+``legacy_json_cast``
+^^^^^^^^^^^^^^^^^^^^^
+
+    * **Type:** ``boolean``
+    * **Default value:** ``true``
+
+    When casting from ``JSON`` to ``ROW``, by default ignore the case of field names in RowType when casting from ``JSON`` to ``ROW`` (for legacy support), the matching would be always case-insensitive.
+    To strictly enforce the case of double quoted field names in RowType when matching, set config property ``legacy_json_cast`` to ``false``. After setting the property, the matching for quoted field name would be case-sensitive, and the matching for unquoted field name remains case-insensitive.
```

Comment:
```suggestion
    
    Set ``legacy_json_cast`` to ``false`` to strictly enforce the case-sensitivity of double quoted field names in ``RowType`` when matching. Matching for unquoted field names remains case-insensitive.
```
Suggest imperative mode for "Set ``legacy_json_cast`` to ``false`` ", and prefacing it with a line break to left-justify this instruction to make it more obvious to the reader what they must do if they want a change in the behavior. Separated "matching for unquoted field name remains case-insensitive" as its own sentence to improve readability. 

You use both "double quoted" and "quoted". Please check my suggestion to make sure that it says what you want it to mean, and correct it as needed.